tests/contracts/callee-1/src/lib.rs

```diff
@@ -5,22 +5,20 @@
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
 #![no_std]
-#![feature(
-    core_intrinsics,
-    lang_items,
-    alloc_error_handler,
-)]
+#![feature(core_intrinsics, lang_items, alloc_error_handler)]
 
 use rkyv::{Archive, Deserialize, Serialize};
 use rusk_uplink::{
     Apply, ContractId, Execute, Query, StoreContext, Transaction,
 };
-use rusk_uplink_derive::{apply, execute, query, state, transaction};
+use rusk_uplink_derive::{apply, execute, init, query, state, transaction};
 
 #[state(new = false)]
 pub struct Callee1State {
     target_address: ContractId,
 }
+#[init]
```

Comment:
This idiom is a bit strange, we should discuss alternatives. I'll open a separate issue though.

Reply:
There's already an issue where the userspace contract API is up for discussion. It would be nice if this was discussed [there](https://github.com/dusk-network/rusk-vm/issues/331).